wrapper/rust/README.md

```diff
@@ -0,0 +1,17 @@
+# wolfSSL Rust Wrapper
+
+## Building the wolfssl Rust Wrapper
+
+First, configure and build wolfssl C library.
+
+Then build the wolfssl Rust wrapper with:
+
+    make -C wrapper/rust
+
```

Comment:
How should wolfSSL be configured and built? Does it matter?

Is there an example of using the rust bindings after created? Something like testwolfcrypt running.